webpay/config_es.xml

```diff
@@ -2,11 +2,12 @@
 <module>
     <name>webpay</name>
     <displayName><![CDATA[Webpay Plus]]></displayName>
-    <version><![CDATA[1.0.0]]></version>
-    <description><![CDATA[Recibe pagos en linea con Tarjetas de crédito y Redcompra en tu Prestashop a través de Webpay Plus REST]]></description>
+    <version><![CDATA[1.0.2]]></version>
```

Comment:
```suggestion
    <version><![CDATA[1.0.0]]></version>
```
This should always be 1.0.0 to the deployment script to work.

Reply:
I think this is should also be 1.0.0

Reply:
done